### ISSUE TITLE
Workflow Approvals doesn't work from Workflow Feed Urls when the Workflow Command has a PowerShell Script Workflow Action

### ISSUE THREAD

**Issue by SubbuRamanathan** [SitecorePowerShell/Console#1204]
### Expected Behavior

Workflow Approvals happening through Workflow Feed Page should work when the associated Workflow Command has a PowerShell Script Workflow Action

### Actual Behavior

Workflow Command with PowerShell Script Workflow Action is throwing error when Approver is trying to approve/reject through Workflow Feed Urls (https://site.local/sitecore/shell/feeds/action.aspx?c=Workflow&id=%7b79902BF1-F289-4B38-A8A1-9DAD2E65AC01%7d&la=en&v=3&cmd=%7b08D7311C-1CEA-4DB3-9466-A24721647071%7d)

### Steps to Reproduce the Problem

**Given:**
Sitecore 9.2.0
SPE 5.1
PowerShell Script Workflow Action added to Workflow Commands(Approve/Reject)
**When:**
Workflow Approver tries to approve/reject through Workflow Feeds(https://doc.sitecore.com/users/92/sitecore-experience-platform/en/subscribe-to-a-client-rss-feed.html#UUID-831051fa-b024-9159-1d69-bd354f1c60a4_id__Subscribe_to_a)
**Then:**
Experiencing this error, 
![image](https://user-images.githubusercontent.com/69419476/89730372-5ba59200-da14-11ea-94ce-73f0387a9e60.png)

**Root Cause:**
While digging deeper, noticed this to be due to default ScriptAction(Cognifide.PowerShell.Integrations.Workflows.ScriptAction, Cognifide.PowerShell) which looks to Broadcast PowershellRunner to ClientPage which isn't available in the Feeds page (action.aspx) as it runs outside of Sitecore Client. Updating the Broadcast Method call by adding the check for Context.ClientPage.Application in Cognifide.PowerShell.Integrations.Workflows.ScriptAction fixed the issue for me.

```
if(Context.ClientPage.Application != null)
{
    Context.ClientPage.ClientResponse.Broadcast(SheerResponse.ShowModalDialog(urlString.ToString(), "400", "220", "PowerShell Script Results", false), "Shell");
}
else
{
    SheerResponse.ShowModalDialog(urlString.ToString(), "400", "220");
}
```

I was able to get this working by creating a new custom ScriptAction class and changing Type String value(ProjectName.Workflows.ScriptAction, AssemblyName) for the required PowerShell Workflow Script Action Items. It would be nicer to have this addressed in the default ScriptAction as Workflow Feed API appears to be the only way to get direct urls for Approve/Reject actions for content approver, so that content approver can approve content updates quickly.

 - [ ] Tested issue with clean install of Sitecore and the latest available version of SPE.
 
 - [ ] Asked questions on the Sitecore Slack Chat channel.
 
 - [x] Reviewed questions and answers on the Sitecore Stack Exchange.

**Comment by michaellwest** [comment]
Thank you for reporting the issues.

A few things that were new to me:
* I've never tried the feed url before
* The feed url does not render as a standard html page until I installed a [chrome extension](https://chrome.google.com/webstore/detail/rss-subscription-extensio/nlbjncdgjeocebhnmkbbbdekmmmcbfjd)

**Comment by michaellwest** [comment]
@SubbuRamanathan  Have not quite figured out how to properly test this. Can you do a new build with the latest version on master and give it a try?

**Comment by SubbuRamanathan** [comment]
Thanks @michaellwest for the quick turnaround,
I will test and confirm back shortly.

**Comment by SubbuRamanathan** [comment]
@michaellwest, I have tested the latest build of master in Sitecore 10. I tried approving items from the different possible entry points for this logic (Top Ribbon Bar - Review tab, Quick Action Bar, Workbox, RSS Feed and Direct URLs) and all of them are working as expected. 

Just FYI, I just noticed that the issue was primarily happening when I submit for approval from Desktop mode and when trying to use Direct Urls or RSS feed Urls(in my case I'm using Direct URLs for integrating Workflow with Microsoft Teams using SPE). RSS Feed Urls and Direct Urls were working when I submit for approval from Content Editor mode, it was weird, seems like it could be because of the way Sitecore triggers the Workflow actions. But with the new code, I'm able to get it working in both cases. I took a quick snapshot of the before/after states for your reference,

**Before**
![Workflow Approval Issue](https://user-images.githubusercontent.com/69419476/90189313-51c4bb80-dd93-11ea-97ec-a982b0328837.gif)

**After**
![Workflow Approval - Working](https://user-images.githubusercontent.com/69419476/90189355-6739e580-dd93-11ea-983c-1ed42436b7ba.gif)

Thank you!
